buffer/src/main/java/io/netty/buffer/ByteBuf.java

```diff
@@ -558,6 +558,17 @@
     public abstract short getShort(int index);
 
     /**
+     * Gets a 16-bit short integer at the specified absolute {@code index} in
+     * this buffer in Little Endian Byte Order. This method does not modify
+     * {@code readerIndex} or {@code writerIndex} of this buffer.
+     *
+     * @throws IndexOutOfBoundsException
+     *         if the specified {@code index} is less than {@code 0} or
+     *         {@code index + 2} is greater than {@code this.capacity}
+     */
+    public abstract short getShortLE(int index);
```

Comment:
Same comment as https://github.com/netty/netty/pull/4477#discussion-diff-45922121. If we want this to go back into 4.0 we would have to make this non-abstract.